src/assets/javascripts/components/toc/index.ts

```diff
@@ -296,7 +296,7 @@ export function mountTableOfContents(
 
       /* Look backward */
       for (const [index, [anchor]] of prev.entries()) {
-        anchor.setAttribute("data-md-state", "blur")
```

Comment:
Need to change `anchor.removeAttribute("data-md-state")` above to `anchor.classList.remove("md-nav__link--passed")`

Reply:
good catch!